tartufo/cli.py

```diff
@@ -158,34 +173,38 @@ def main(ctx: click.Context, **kwargs: config.OptionTypes) -> None:
     also be made to work in pre-commit mode, for scanning blobs of text as a
     pre-commit hook.
     """
+
     options = types.GlobalOptions(**kwargs)  # type: ignore
     ctx.obj = options
+    if options.quiet and options.verbose > 0:
+        raise click.BadParameter("-v/--verbose and -q/--quiet are mutually exclusive.")
 
 
 @main.resultcallback()  # type: ignore
 @click.pass_context
 def process_issues(
     ctx: click.Context,
-    result: Tuple[str, List[scanner.Issue]],
+    result: Tuple[str, scanner.ScannerBase],
     **kwargs: config.OptionTypes,
 ):
-    repo_path, issues = result
+    repo_path, scan = result
```

Comment:
I think I'd prefer if this was called `scanner`, just to be more explicit about what it actually is. Not a blocker though!

Reply:
`scanner` shadows the import name scanner, so I landed on `scan` as name for a scanner instance